css/mystyle.css

```diff
@@ -0,0 +1,32 @@
+form {
+  background-color: lightblue;
+  width: 25%;
+  display: flex;
+  justify-content: center;
+  flex-direction: column;
+  align-items: center;
+  border: 1px solid black;
+}
+
+.form-input {
+	height: 25px;
+	margin: 10px;
```

Comment:
Is the different indentation here on purpose? Friendly reminder: IDEs are pretty useful tools. They have shortcuts. You may like using one.